tests/e2e/nvidia-container-toolkit_test.go

```diff
@@ -234,12 +240,28 @@ var _ = Describe("docker", Ordered, ContinueOnFailure, func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(output).To(Equal("ModifyDeviceFiles: 0\n"))
 		})
+
+		//sudo docker run --runtime=nvidia --rm -ti -e NVIDIA_VISIBLE_DEVICES=all -e NVIDIA_DRIVER_CAPABILITIES=all --mount type=bind,source=/tmp/empty,target=/empty,bind-propagation=shared  ubuntu true
```

Comment:
What is the purpose of this comment?

Reply:
oh, sorry it was me during devel, to not forget that bit